### ISSUE TITLE
Generating map<K, V> on javascript type issue 

### ISSUE THREAD

**Issue by sayjeyhi** [protocolbuffers/protobuf-javascript#14]

**Comment by dibenede** [comment]
Agreed that we could generate a better type here. As far as we can tell, grpc-web is just generating the type reflecting what we do.

In the meantime, if you call getMetadataMap() you should get a Map type back.
